components/merkledb/benches/schema_patterns.rs

```diff
@@ -13,17 +13,15 @@
 // limitations under the License.
 
 use criterion::{black_box, Bencher, Benchmark, Criterion, Throughput};
-use exonum_derive::FromAccess;
+use exonum_derive::*;
```

Comment:
Nit/style: I don't really like wildcard imports, IMHO specifying concrete imports really helps when you try to get what's going on in the module seeing it first time (or after a long time).